### PR TITLE
Add failing test for i64 arithmetic

### PR DESCRIPTION
**Pull request by xtuc**
Refs #22 

:dancer:

### CODE REVIEW

File: test/interpreter/fixtures/64bits-arithmetic/module.wast [hunk 1]
```diff
@@ -0,0 +1,10 @@
+(module
+  (func $test (result i32) (
```
Comment (xtuc) [inline test/interpreter/fixtures/64bits-arithmetic/module.wast]: Note that I could use i64 in the signature because of #21

**Comment by ColinEberhardt** [comment]
Unfortunately the wast module in this PR doesn't compile:

```
$ wat2wasm test/interpreter/fixtures/64bit-arithmetic/module.wast
test/interpreter/fixtures/64bit-arithmetic/module.wast:7:6: error: type mismatch in implicit return, expected i32 but got i64.
    (i64.sub)
```

**Comment by xtuc** [comment]
Well that makes sense to me:
- the wast is still using an invalid syntax, which I refactored here  https://github.com/xtuc/js-webassembly-interpreter/pull/52
- as mentioned in the error, the function returns an i32 but an i64 was left on the stack. We could implement the conversions to make it work https://github.com/xtuc/js-webassembly-interpreter/issues/63

Sorry about that. We can also add stricter validations like wabt does to avoid that in the future.

**Comment by ColinEberhardt** [comment]
No worries.

I’ve implemented a very crude prototype of i64, that uses long.js. I also came up with the idea of using a cast to to verify in the tests. 

I’ll tidy up a bit and create a PR.

**Comment by xtuc** [comment]
Closing this since it was basically included in https://github.com/xtuc/js-webassembly-interpreter/pull/79

**Pull request closed without merging**